src/components/Global/icons/svg/ScaleSVG.js

```diff
@@ -0,0 +1,26 @@
+import React from "react";
```

Comment:
Since there are multiple of these SVG components. It might be fun to build a generator that can make these files/components for us. We might just have a function that we feed the name, path and default dimensions and then it would create a component for us. Just a thought!